microsoft-edge/webview2/release-notes.md

```diff
@@ -122,89 +122,145 @@ The following items are now stable:
 
-<!-- ## 1.0.1549-prerelease
+## 1.0.1619-prerelease
 
-Release Date: December 12, 2022
+Release Date: January 19, 2023
 
-[NuGet package for WebView2 SDK 1.0.1549-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1549-prerelease)
+[NuGet package for WebView2 SDK 1.0.1619-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1619-prerelease)
 
-For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 110.0.1549.0 or higher.
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 111.0.1619.0 or higher.
 
 ### General
 
 #### Experimental features 
 
-*  Added support for the Locale Region API:
+*  Added support for the Permission management API:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* * CoreWebView2PermissionRequestedEventArgs Class
+   CoreWebView2PermissionRequestedEventArgs.SaveInProfile Property
+* ICoreWebView2PermissionSetting Class
+   ICoreWebView2PermissionSetting.PermissionKind Property
+   ICoreWebView2PermissionSetting.PermissionOrigin Property
+   ICoreWebView2PermissionSetting.PermissionState Property
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 
-* 
+* CoreWebView2PermissionRequestedEventArgs Class
+   CoreWebView2PermissionRequestedEventArgs.SaveInProfile Property
+* ICoreWebView2PermissionSetting Class
+   ICoreWebView2PermissionSetting.PermissionKind Property
+   ICoreWebView2PermissionSetting.PermissionOrigin Property
+   ICoreWebView2PermissionSetting.PermissionState Property
 
 ##### [Win32/C++](#tab/win32cpp)
 
-* 
+* ICoreWebView2ExperimentalPermissionRequestedEventArgs3
+   get_SavesInProfile
+   put_SavesInProfile
+* ICoreWebView2ExperimentalSetPermissionStateCompletedHandler
+* ICoreWebView2ExperimentalGetNonDefaultPermissionSettingsCompletedHandler
+* ICoreWebView2ExperimentalProfile6
+   SetPermissionState
+   GetNonDefaultPermissionSettings
+* ICoreWebView2ExperimentalPermissionSettingCollectionView
+   GetValueAtIndex
+   get_Count
+* ICoreWebView2ExperimentalPermissionSetting
+   get_PermissionKind
+   get_PermissionOrigin
+   get_PermissionState
 
 ---
 
-*  Added support for the tracking prevention API:
+*  Added support for API to disable back and forward navigation:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* CoreWebView2NavigationStartingEventArgs
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 
-* 
+* CoreWebView2NavigationStartingEventArgs
 
 ##### [Win32/C++](#tab/win32cpp)
 
-* 
-        
+* ICoreWebView2ExperimentalNavigationStartingEventArgs2
+
 ---
 
 #### Promotions
 
 The following APIs are promoted to stable in this prerelease SDK:
 
-*  Added support for the Print API:
+*  The Custom Scheme Registration API:
 
 ##### [.NET/C#](#tab/dotnetcsharp)
 
-* 
+* CoreWebView2EnvironmentOptions Class
+   CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property
+* CoreWebView2CustomSchemeRegistration Class
 
 ##### [WinRT/C#](#tab/winrtcsharp)
 

```

Comment:
@nishitha-burman @peiche-jessica - property not found, for WinRT:
CoreWebView2EnvironmentOptions.**CustomSchemeRegistrations** Property

Class:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environmentoptions?view=webview2-winrt-1.0.1619-prerelease

It's found for .NET:
* [CoreWebView2EnvironmentOptions.CustomSchemeRegistrations Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.customschemeregistrations?view=webview2-dotnet-1.0.1619-prerelease&preserve-view=true)

Equiv is found for Win32:
   * [ICoreWebView2EnvironmentOptions4::GetCustomSchemeRegistrations method](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.1619-prerelease&preserve-view=true#getcustomschemeregistrations)
   * [ICoreWebView2EnvironmentOptions4::SetCustomSchemeRegistrations method](https://learn.microsoft.com//microsoft-edge/webview2/reference/win32/icorewebview2environmentoptions4?view=webview2-1.0.1619-prerelease&